packages/ckeditor5-image/src/image/imagetypetogglecommand.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module image/image/imagetypetogglecommand
+ */
+
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import { insertImage, isImage, isImageInline } from './utils';
+
+/**
+ * The image inline command. It is used to convert block image io inline image.
+ *
+ * @extends module:core/command~Command
+ */
+export default class ImageTypeToggleCommand extends Command {
+	/**
+	 * @inheritDoc
+	 */
+	refresh() {
+		const element = this.editor.model.document.selection.getSelectedElement();
+
+		this.isEnabled = isImage( element ) || isImageInline( element );
+	}
+
+	/**
+	 * Executes the command.
+	 *
+	 * @fires execute
+	 */
```

Comment:
This is less descriptive than in the base class so maybe it should be `@inheritDoc` ?